### DISCUSSION TITLE
Prevent a XML file in the public folder from being wrapped in HTML

### DISCUSSION THREAD

**Discussion by Theeeus** [vercel/next.js#48268] [Help] [answered]
### Summary

I've created a sitemap.xml file and added it to the public folder of my project. However, when I access it and inspect the code I can see it's wrapped in HTML. For that same reason Google is not accepting the URL saying it's in HTML instead of XML.

So the question is: How to ensure the file is shown in XML instead of HTML?

### Example

https://www.surfphotosmate.com.au/sitemap.xml

**Comment by wwwdepot** [comment, marked as answer]
Your problem is the `hasAccessedBefore` cookie.
